app/core/rate_limiting/schemas.py

```
@@ -0,0 +1,68 @@
+"""
+Schemas for rate limiting data structures.
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)